README.md

```diff
@@ -56,8 +56,8 @@ Gazebo Classic and Ignition software.
     1. `cd <path to source code>`
     1. Make sure you're in the branch to be released, after the pull request bumping the version has been merged.
     1. Dry-run `release.py` with the appropriate arguments. Some examples:
-        1. New Ignition minor release: `<path to release-tools>/release.py -r 1 ign-physics3 3.1.0 --ignition-repo -a ignition-physics3 secrettoken --dry-run`
-        1. Pre-release: `<path to release-tools>/release.py -r 1 ign-physics3 3.0.0~pre1 --ignition-repo -a ignition-physics3 --upload-to-repo=prerelease secrettoken --dry-run`
+        1. New Ignition minor release: `<path to release-tools>/release.py -r 1 ign-physics3 3.1.0 secrettoken --dry-run`
+        1. Pre-release: `<path to release-tools>/release.py -r 1 ign-physics3 3.0.0~pre1 --upload-to-repo=prerelease secrettoken --dry-run`
         1. Classic release: `<path to release-tools>/release.py -r 1 gazebo11 11.2.0 secrettoken --dry-run`
```

Comment:
nit: I think the 1st and 3rd sub-items are now redundant and could be combined ("New Ignition minor release" and "Classic release")

Reply:
200d153